biblatex.cfg

```diff
@@ -0,0 +1,20 @@
+%Configure biblatex to fit ISO-690 UTBM's takes
+%Expect biblatex style to be authortitle as iso-authoryear force italic for title
+\DefineBibliographyStrings{french}{% Define words to prefix url and urldate
+  urlseen = {consulté le},
+  urlfrom = {disponible sur},
+}
```

Comment:
Ça marche aussi en anglais ou faut déclarer les strings aussi ? 👀

Reply:
ça marche tant que t'as ton babel qui est set en french quelque soit la langue de la source , tu peut mettre les même string pour d'autre language au besoin

Reply:
Ça serait possible de le faire pour l'anglais ? 👉🏻👈🏻

Reply:
``\DefineBibliographyStrings{english}{% Define words to prefix url and urldate
  urlseen = {last seen the},
  urlfrom = {accessible on},
}
cado xD